addons/core/addon/components/form/authenticate/index.hbs

```diff
@@ -1,3 +1,5 @@
+{{!-- //TODO fix the issue and delete disabling the rule --}}
+{{!-- template-lint-disable no-passed-in-event-handlers --}}
```

Comment:
I had to skip this rule in this file, it did not like we are passing `@submit` as arguments, [more infor about it here](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-passed-in-event-handlers.md)